docs/modules/agent/overview.md

```diff
@@ -25,6 +25,21 @@ User-configured path monitoring, able to monitor the created file information
 Directory regular filtering, support YYYYMMDD+regular expression path configuration
 Breakpoint retransmission, when InLong-Agent restarts, it can automatically re-read from the last read position to ensure no reread or missed reading.
 
+#### File options
+| Parameter                     | Required | Default value | Type   | Description                                                  |
+| ----------------------------- | -------- | ------------- | ------ | ------------------------------------------------------------ |
+| pattern                       | required | (none)        | String | File pattern. For example: /root/[*].log      |
+| timeOffset                    | optional | (none)        | String | File name includes time, for example: *** YYYYMMDDHH *** YYYY represents the year, MM represents the month, DD represents the day, and HH represents the hour, *** is any character. '1m' means one minute after, '-1m' means one minute before. '1h' means one hour after, '-1h' means one hour before. '1d' means one day after, '-1d' means one day before.|
```

Comment:
*** YYYYMMDDHH ***  should be ` YYYYMMDDHH `  ？

Reply:
Please correct the above format errors in the Chinese document in the same time.